src/Trip/Trip.ts

```diff
@@ -87,6 +87,7 @@ export interface ItineraryItem {
 	startTime: number | null; // Number of seconds from midnight.
 	duration: number | null; // Time in seconds planned to spend visiting place.
 	note: string | null;
+	isSticky: boolean | null; // https://confluence.sygic.com/display/STV/Sticky+Places+in+Itinerary
```

Comment:
It can not be set properly without knowing context of whole trip and maybe some related user data (in future). It is set after whole trip is built. I've chosen the default value null, so that we have information that it was not set yet and therefore we can not say if it is or isn't sticky.